tor/core/user_interaction.py

```diff
@@ -88,7 +88,10 @@ def process_claim(post, config):
 
         if flair.unclaimed in top_parent.link_flair_text:
             # need to get that "Summoned - Unclaimed" in there too
-            post.reply(_(claim_success))
+            if first_claim:
+                post.reply(_(claim_success.format(first_claim_success)))
+            else:
+                post.reply(_(claim_success.format(''))
 
```

Comment:
I’m not a huge fan of shoving one template into another like this since it encourages placing too much logic in a string formatted when it would probably fit better in a method call that follows a string builder pattern.

In this case, though, I think it’s simple enough that it might be clearer to have some duplication. Perhaps we could have 2 separate strings be used for the messages?

```python
if first_claim:
    post.reply(_(first_claim_success))
else:
    post.reply(_(claim_success))
```

Reply:
It was @itsthejoker decision to switch to this 'shoving' method. What do you think Joe?

Reply:
To be clear, we should be keeping the first-claim messaging almost the same as the subsequent claim messages, but I'm not sure we should be enforcing that with code. That's more of an opinion thing though. Also interested in hearing Joe's thoughts on the matter.

Reply:
@itsthejoker ping

Reply:
Ugh, sorry. I personally prefer logic-ing the templates because of DRY. I'd rather update one template and not have to worry about it than updating two. That said, this is short enough that it's probably not an issue.

Reply:
You know, I'm not gonna [bikeshed](https://en.wiktionary.org/wiki/bikeshedding) this. It's low enough importance that we can just keep it as-is. 👍